services/provider/server/server.go

```diff
@@ -320,7 +321,7 @@ func (s *OCSProviderServer) getExternalResources(ctx context.Context, consumerRe
 	for _, i := range consumerResource.Status.CephResources {
 		switch i.Kind {
 		case "CephClient":
-			clientSecretName, err := s.getCephClientSecretName(ctx, i.Name)
+			clientSecretName, annotation, err := s.getCephClientSecretName(ctx, i.Name)
```

Comment:
Why is this var called annotation? it is taken from an annotation but it is not an annotation. 
Why not just call it cephUserType as suggested before?

Reply:
now it look something like this `clientSecretName, cephClaim, cephUserType, err := s.getCephClientInformation(ctx, i.Name)`